ci/long_running_tests/workloads/serve_scalability.py

```diff
@@ -0,0 +1,117 @@
+# A multi-node scalability test. We put an http proxy on the head node and spin
```

Comment:
this also doesn't belong to long running test. the expectation is that long running test will keep running forever if needed.